minesweeper.py

```diff
@@ -224,6 +219,13 @@ def flagging(event):
 )
 flag_image = PhotoImage(file=(where_this_file_is / "flag_small.png"))
 bomb_image = PhotoImage(file=(where_this_file_is / "bomb_small.png"))
+frames = [
+    PhotoImage(
+        file=where_this_file_is / "doomguy.gif",
+        format="gif -index %i" % (i),
```

Comment:
```suggestion
        format=f"gif -index {i}",
```

Not a big deal, but we might as well use f-strings consistently.